redaxo/src/core/lib/autoload.php

```diff
@@ -102,7 +102,10 @@ public static function autoload($class)
 
         // Class not found, so reanalyse all directories if not already done or if $force==true
         // but only if an admin is logged in
-        if ((!self::$reloaded || $force) && ($user = rex_backend_login::createUser()) && $user->isAdmin()) {
+        if (
+            (!self::$reloaded || $force) &&
+            (rex::isDebugMode() || ($user = rex_backend_login::createUser()) && $user->isAdmin())
```

Comment:
Ich bin gerade an #1099 dran. Dabei hatte ich gerade den Fall, dass eine neue Klasse nicht in der Konsole nicht gefunden wurde.
Dann dachte ich mir: Klar, ist ja auch kein User vorhanden. Daher forciere ich mal den Debug-Modus in der Console-Application, erschien mir auch sowieso sinnvoll.
Und dann habe ich mich gewundert, dass die Klasse trotzdem weiterhin nicht gefunden wurde, dachte eigentlich das wäre bereits so. Daher nun der Vorschlag.

Also kurzum: Ich wollte dem Entwickler in der Console den gleichen Komfort bieten, wie wenn man im Backend als Admin eingeloggt ist. Dass neue Klassen automatisch gefunden werden und man den Cache nicht löschen braucht.

Sicher bin ich mir hier aber auch nicht.